ui/src/app/store/fabric/slice.ts

```diff
@@ -69,7 +69,8 @@ const fabricSlice = createSlice({
           method: "set_active",
         },
         payload: {
-          params: { [FabricMeta.PK]: id },
+          // Server unsets active item if primary key is not sent.
+          params: id === null ? null : { [FabricMeta.PK]: id },
```

Comment:
Accidentally screwed up when I copied these over from elsewhere - the entire `params` object needs to be null, not just the id.